clang/include/clang/Lex/Preprocessor.h

```diff
@@ -136,6 +137,64 @@ struct CXXStandardLibraryVersionInfo {
   std::uint64_t Version;
 };
 
+/// Record the previous 'export' keyword info.
+//
+// Since P1857R3, the standard introduced several rules to determine whether the
+/// 'module', 'export module', 'import', 'export import' is a valid
+/// directive introducer. This class used to record the previous 'export'
+/// keyword token, and then handle 'export module' and 'export import'.
```

Comment:
```suggestion
// Since P1857R3, the standard introduced several rules to determine whether the
/// 'module', 'export module', 'import', 'export import' is a valid
/// directive introducer. This class is used to record the previous 'export'
/// keyword token, and then handle 'export module' and 'export import'.
```